ci/check-documentation.py

```diff
@@ -86,6 +87,31 @@ def complain(message):
         # check local link validity
 
         for url in sorted([ref.url for ref in doc_page.external_references()]):
+            if url.startswith(omz_github_url):
+                omz_relative_url = url[len(omz_github_url):]
+                omz_relative_path = Path(urllib.request.url2pathname(omz_relative_url))
+
+                if omz_relative_path.as_posix().startswith(('blob/', 'tree/')):
+                    omz_relative_path = Path(*omz_relative_path.parts[1:]) # removed blob/ or tree/
+                    omz_relative_posix_path = omz_relative_path.as_posix()
+
+                    distance_to_md_parent_dir = 0
+
+                    if url.endswith('.md'):
+                        for parent in md_path_rel.parents:
+                            try:
+                                omz_relative_posix_path = omz_relative_path.relative_to(parent).as_posix()
+                                break
+                            except ValueError:
+                                distance_to_md_parent_dir += 1
+                        suggested_path = '../' * distance_to_md_parent_dir + omz_relative_posix_path
```

Comment:
This value should be converted back to a URL path (with `pathname2url`), since it's going to be used as the link target.